.buildkite/scripts/pipelines/security_solution_quality_gate/upload_runtime_info.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+echo "$KIBANA_DOCKER_PASSWORD" | docker login -u "$KIBANA_DOCKER_USERNAME" --password-stdin docker.elastic.co
+
+if [ "$KIBANA_CURRENT" = "1" ]; then
+    docker pull docker.elastic.co/kibana-ci/kibana-serverless:git-${BUILDKITE_COMMIT:0:12}
+    build_date=$(docker inspect docker.elastic.co/kibana-ci/kibana-serverless:git-${BUILDKITE_COMMIT:0:12} | jq -r '.[0].Config.Labels."org.label-schema.build-date"')
```

Comment:
What do you think about extracting some of these expressions to variables (like in `create_docker_image.sh`)? Or these all could be using the same `KIBANA_IMAGE` variable basically, that's the one moving part in all these commands

Reply:
Refactored towards the proposed direction. Please have a look.